USAGE.md

```diff
@@ -54,15 +100,15 @@ This step provisions some identity and group test data, including:
 
 Booting the [server components](#5-Access-the-Playground) might take a while. Verify that all server components are started before bootstrapping:
 ```bash
-docker-compose logs -f ipa
-docker-compose logs -f keycloak
-docker-compose logs -f vault
-docker-compose logs -f k3s-server
+sudo ${CONTAINER_RUNTIME}-compose logs -f ipa
```

Comment:
Why `sudo`?

Reply:
in podman when running commands, they're being run in a completely different context if run either as a user or with sudo.